applicationinsights/applicationinsights-js.d.ts

```diff
@@ -0,0 +1,488 @@
+// Type definitions for ApplicationInsights-JS v0.22.14
+// Project: https://github.com/Microsoft/ApplicationInsights-JS
```

Comment:
Is this right? not https://github.com/Microsoft/ApplicationInsights-node.js ?
https://www.npmjs.com/package/applicationinsights